app/src/main/java/com/sunlightlabs/congress/services/RollService.java

```diff
@@ -9,29 +11,67 @@
 import org.json.JSONObject;
 
 import java.text.ParseException;
+import java.text.SimpleDateFormat;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
 
 public class RollService {
-	
-	public static String[] basicFields = {
+
+    public static String datetimeFormat = "yyyy-MM-dd hh:mm:ss";
+
+    // standard field names that map to Roll vote type names
+    public static final String YEA_FIELD = "yes";
+    public static final String NAY_FIELD = "no";
+    public static final String PRESENT_FIELD = "present";
+    public static final String NOT_VOTING_FIELD = "not_voting";
+
+    // standard response in Pro Publica API for vote positions
+    public static final String YEA = "Yes";
+    public static final String NAY = "No";
+    public static final String PRESENT = "Present";
+    public static final String NOT_VOTING = "Not Voting";
+
+    // Pro Publica API uses "Speaker" to represent Speaker not voting
+    public static final String SPEAKER = "Speaker";
+
+    // Pro Publica API uses these to denote certain House vote types
+    public static final String VOTE_HALF_ONE = "YEA-AND-NAY";
+    public static final String VOTE_HALF_TWO = "RECORDED VOTE";
+    public static final String VOTE_TWO_THIRDS  = "2/3 YEA-AND-NAY";
```

Comment:
@dwillis These are all the vote types I could find. Let me know if my assumptions here are incorrect - I'm assuming that `YEA-AND-NAY` and `RECORDED VOTE` are always `1/2`, and that those plus `2/3 YEA-AND-NAY` are all the vote types we could expect in the House.